exporter/awsemfexporter/grouped_metric.go

```diff
@@ -17,6 +17,8 @@ package awsemfexporter
 import (
 	"go.opentelemetry.io/collector/model/pdata"
 	"go.uber.org/zap"
+	"encoding/json"
```

Comment:
Think these aren't grouped correctly and other files have wrong ordering, run the formatting on the source to fix the build